Update ChromeOS installation instructions (#8304)
flutter/website: src/get-started/install/_chromeos-android-sdk-setup.md

$ sudo apt install default-jdk
```

### Install the Android SDKs

Download the [Android SDK tools][] and
select the “Command Line Tools only” option.

Drag and drop the downloaded zip into your Linux Files folder through the

> ChromeOS Files app. This moves it to the home directory, notated as
> $TOOLS_PATH going forward (`~/`).

Reviewer: ```suggestion
ChromeOS Files app. This moves the file to the home directory,
and is referred to as $TOOLS_PATH going forward (`~/`).
```
